config/config_params.go

```diff
@@ -225,6 +225,8 @@ type Config struct {
 
 	XDPEnabled        bool `config:"bool;true"`
 	GenericXDPEnabled bool `config:"bool;false"`
+
+	NATOutgoingAddress net.IP `config:"ipv4;"`
```

Comment:
Suggest moving this up to line 200 since it naturally groups with `NATPortRange`